client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -124,6 +133,31 @@ export default function CompositeCheckout( {
 		[ allowedPaymentMethods, isLoading, isLoadingStoredCards, storedCards, registerStore ]
 	);
 
+	const validateDomainContact =
+		validateDomainContactDetails || wpcomValidateDomainContactInformation;
+
+	const renderDomainContactFields = (
+		domainNames,
+		contactDetails,
+		updateContactDetails,
+		applyDomainContactValidationResults
+	) => {
+		return (
+			<WPCheckoutErrorBoundary componentTitle="ContactDetailsFormFields">
```

Comment:
What is the `componentTitle` prop for?

Reply:
Good catch- it is from an earlier iteration of WPCheckoutErrorBoundary that tried to be a little too clever, and is no longer needed.